README.md

```diff
@@ -1,5 +1,4 @@
-![LOG](logo.png)
-
+![image](https://user-images.githubusercontent.com/20048096/135674268-018ff2bc-f3e8-4ae1-b19a-c7cd1eb1a519.png)
```

Comment:
It would be better to replace the file 'logo.png' in the top-level directory for this project.  Then no change to the README.md is needed.